### PR TITLE
EJDOTNETCORE-3482-Update SkiaSharp NuGet information for Word to PDF in Portable platforms

### PR DESCRIPTION
**Pull request by AntonihilSahayaraj**
-Update SkiaSharp NuGet information for Word to PDF in Portable platforms

### CODE REVIEW

File: File-Formats/DocIO/NuGet-Packages-Required.md [hunk 1]
```diff
@@ -266,7 +266,7 @@ Install-Package Syncfusion.DocIORenderer.NET
...
 N> 2. Syncfusion components are available in [nuget.org](https://www.nuget.org/)
 N> 3. Please refer the procedure to deploy your .NET Core application in Linux OS from [here](https://www.syncfusion.com/kb/8470/how-to-deploy-net-core-application-with-word-to-pdf-conversion-capabilities-in-linux-os).
-N> 4. From v18.4.0.x, the dependent package SkiaSharp is upgraded from 1.59.3 to 2.80.2 version and it is mandatory to use SkiaSharp.NativeAssets.Linux v2.80.2 package instead of SkiaSharp.Linux v1.59.3 for converting Word documents into PDF in Linux environment.
+N> 4. From v20.1.0.x, the dependent package SkiaSharp is upgraded from 2.80.2 to 2.88.0-preview.209 version and it is mandatory to use SkiaSharp.NativeAssets.Linux v2.88.0-preview.209 package for converting Word documents into PDF in Linux environment.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/NuGet-Packages-Required.md:269]: From v20.1.0.x, the dependent package SkiaSharp is upgraded from 2.80.2 to 2.88.0-preview.209 version and it is mandatory to use SkiaSharp.NativeAssets.Linux v2.88.0-preview.209 andHarfBuzzSharp.NativeAssets.Linux v2.8.2-preview.209 packages for converting Word documents into PDF in Linux environment.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/NuGet-Packages-Required.md (hunk 1 above)
Comment (AntonihilSahayaraj) [reply File-Formats/DocIO/NuGet-Packages-Required.md]: As per your feedback, i have changed the content

**Comment by AntonihilSahayaraj** [comment]
rebuild

**Comment by DeepakRajSundar** [comment]
rebuild

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**